contrib/kube-prometheus/jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -100,6 +104,25 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
           baseImage: $._config.imageRepos.alertmanager,
           nodeSelector: { 'beta.kubernetes.io/os': 'linux' },
           serviceAccountName: 'alertmanager-' + $._config.alertmanager.name,
+
+          affinity: {
```

Comment:
Although I understand why we would want this by default, the generated manifests now result in something that won't schedule on minikube.

We should just make this a mixin, so users can just do

```
(import "kube-prometheus.libsonnet") +
(import "kube-prometheus-anti-affinity.libsonnet)
```